src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
```

Comment:
it would be great to use constants for such numbers

```suggestion
  static DEFAULT_HEALTH = 100;

  constructor(name, health = DEFAULT_HEALTH) {
```